api_practice/database.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import create_engine
```

Comment:
File to be deleted. Not relevant for socialapi

Reply:
file deleted